README.md

```diff
@@ -1,43 +1,31 @@
-# Structured Data Demo
+# Structured Results Demo
 
 This is a sample project implementing a full page search with structured data results using [Algolia](https://www.algolia.com).
 
-![Structured Data](./structured_data.gif)
+Learn more on the [Structure Results](https://www.algolia.com/doc/guides/building-search-ui/resources/ui-and-ux-patterns/tutorials/structured-results/js/) guide.
```

Comment:
since this is linked, but the doc is still old, do we need to merge both at the same time?

Reply:
Yes